infrastructure/cloudtrail.tf

```diff
@@ -1,17 +1,7 @@
 
-resource "aws_cloudwatch_log_group" "cloudtrail" {
-  name              = "crossfeed-${var.stage}-cloudtrail-logs"
-  retention_in_days = 3653
-  kms_key_id        = aws_kms_key.key.arn
-  tags = {
-    Project = var.project
-    Stage   = var.stage
-  }
-}
-
 resource "aws_cloudtrail" "all-events" {
   name                       = "all-events"
-  s3_bucket_name             = "crossfeed-${var.stage}-cloudtrail-logs"
+  s3_bucket_name             = "cisa-crossfeed-${var.stage}-cloudtrail"
```

Comment:
Should this be equal to `var.cloudtrail_bucket_name`?

Reply:
The names should match with this logic, but I agree, your suggestion makes more sense. I'll implement that in the next PR.